packages/clerk-js/src/core/auth/cookies/devBrowser.ts

```diff
@@ -1,5 +1,6 @@
 import { createCookieHandler } from '@clerk/shared/cookie';
 import { addYears } from '@clerk/shared/date';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
❓ Why we are disabling the eslint rule here, seems that it's not related to the PR